src/BinaryParsers/ElfBinary/ElfBinary.cs

```diff
@@ -57,13 +57,13 @@ public ElfBinary(Uri uri, string localSymbolDirectories = null, bool forceCompre
                 SectionRegions = ELF.Sections.Where(s => s.LoadAddress > 0).OrderBy(s => s.LoadAddress).ToArray();
 
                 CompilationUnits = new Lazy<List<DwarfCompilationUnit>>(()
-                    => DwarfSymbolProvider.ParseAllCompilationUnits(this,
+                    => LoadDebug(DwarfSymbolProvider.ParseAllCompilationUnits(this,
```

Comment:
>ParseAllCompilationUnits [](http://example.com/codeflow?start=53&length=24)

You have broken the formatting for these parameters, each of which should have its own line, each of which should be left-aligned with the first parameter.